executor/insert_common.go

```diff
@@ -74,7 +74,10 @@ type InsertValues struct {
 }
 
 type defaultVal struct {
+	// common default value is a constant.
 	val types.Datum
+	// default expr like `nextval(seq)` should be evaluated row by row.
```

Comment:
How are other default expressions implemented? e.g. current_timestamp().

Reply:
insert into (now()),(now())... only eval once, treated as constant.

Reply:
The comment format is like `expr(variable name) ...`.